src/pages/StorageScreen.tsx

```diff
@@ -13,92 +12,86 @@ import { humanFileSize } from '@/utils/filesize';
 const BAR_WIDTH = window.innerWidth / 3;
 const BAR_HEIGHT = 25;
 
-const StockageScreen = (): JSX.Element => {
+const StorageBar = () => {
   const theme = useTheme();
-  const { t } = useAccountTranslation();
   const { t: translateMessages } = useMessagesTranslation();
+  const { data: storage, isLoading } = hooks.useMemberStorage();
+  if (storage) {
+    const { current, maximum } = storage;
 
-  const { data: stockage, isLoading } = hooks.useMemberStorage();
-
-  const renderComponent = () => {
-    if (stockage) {
-      const { current, maximum } = stockage;
-
-      return (
-        <>
-          <Stack direction="row" alignItems="center">
-            <Stack>
-              <div
-                style={{
-                  background: theme.palette.primary.main,
-                  width: Math.min((current / maximum) * BAR_WIDTH, BAR_WIDTH),
-                  height: BAR_HEIGHT,
-                }}
-              />
-            </Stack>
-            <Stack>
-              <div
-                style={{
-                  background: 'lightgrey',
-                  width: BAR_WIDTH - (current / maximum) * BAR_WIDTH,
-                  height: BAR_HEIGHT,
-                }}
-              />
-            </Stack>
+    return (
+      <>
+        <Stack direction="row" alignItems="center">
+          <Stack>
+            <div
+              style={{
+                background: theme.palette.primary.main,
+                width: Math.min((current / maximum) * BAR_WIDTH, BAR_WIDTH),
```

Comment:
Maybe it could be interesting to export the storage progression in a const to have a better understanding on the calcul and to reuse it in the other div ?

```const storageUsage = (current / maximum) * BAR_WIDTH;```

```suggestion
                width: Math.min(storageUsage, BAR_WIDTH),
```